### PR TITLE
update README

### PR DESCRIPTION
**Pull request by hougangliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/281)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,12 +61,12 @@ In the Study config file, we define the feasible space of parameters and configu
 
 - name: Study name
 - owner: Owner
-- objectivevaluename: Name of the objective value. Your evaluated software should be print log `{objectivevaluename}={objective value}` in std-io.
-- optimizationtype: Optimization direction of the objective value. 1=maximize 2=minimize
-- suggestalgorithm: [random, grid, hyperband] now
+- objectivevaluename: Name of the objective value. Your evaluated software should print log `{objectivevaluename}={objective value}` in std-io.
+- optimizationtype: Optimization direction of the objective value. [minimize, maximize] now.
```
Comment (johnugeorge) [inline README.md]: now ?

Comment (hougangliu) [reply README.md]: updated to **[minimize, maximize] are available now**

**Comment by richardsliu** [comment]
Do you want to also update the README for TFJobs? It is supported now.

**Comment by hougangliu** [comment]
OK, I will try tfjob right now and update README after then.

**Comment by hougangliu** [comment]
It seems **StudyConfig** section is out of date( I think CRD StudyJob had replaced it now). I think we should remove the section and detail how to define StudyJob in another user doc.

**Comment by richardsliu** [comment]
@hougangliu Sounds good, please remove it.

**Comment by hougangliu** [comment]
@richardsliu Thanks! Done!

**Pull request merged**